va/va_cp_h264.h

```diff
@@ -0,0 +1,481 @@
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t long_term_frame_idx[17];
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t va_reserved[VA_PADDING_MEDIUM];
+}VACencDecRefPictureMarkingBufferH264;
+
+/**
+ * \brief Slice parameter for H.264 cenc decode in baseline, main & high profiles.
+ *
+ * This structure holds information for \c
+ * slice_layer_without_partitioning_rbsp() and nal_unit()of the slice 
+ * as defined by the H.264 specification.
+ *
+ */
+typedef struct _VACencSliceParameterBufferH264 {
+
+     /** \brief Slice type.
+     *  Range: 0..2, 5..7, i.e. no switching slices.
+     */
+    uint8_t slice_type;
+     /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t nal_ref_idc;
+    /** \brief Indicate if this is coded slice of an IDR picture. 
+    * Same as the H.264 bitstream syntax element.*/
+    uint8_t idr_flag;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint32_t frame_number;
+    /** \brief Indicate if this is a field or frame picture. */
+    uint8_t field_frame_flag;
+    /** \brief top field or frame POC. */
+    int32_t top_field_order_cnt;
+     /** \brief bottom field POC. */
+    int32_t bottom_field_order_cnt;
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t va_reserved[VA_PADDING_MEDIUM];
+} VACencSliceParameterBufferH264;
+
+/* field_frame_flag in VACencSliceParameterBufferH264 could be the following */
+#define VA_CENC_PICTURE_H264_TOP_FIELD         1
+#define VA_CENC_PICTURE_H264_BOTTOM_FIELD      2
+#define VA_CENC_PICTURE_H264_FRAME             3
+#define VA_CENC_PICTURE_H264_INVALID           4
+
+/**
+ * \brief SEI parameter for h.264 cenc decode in baseline, main & high profiles.
+ *
+ * This structure holds information for \c sei_payload() as
+ * defined by the H.264 specification. 
+ * Only includes buffering_period() payload, pic_timing()payload
+ * and recovery_point()
+ */
+typedef struct _VACencSeiParameterBufferH264 {
+    /** @name sei buffering period*/
+    /**@{*/
+    /** \brief Same as the H.264 bitstream syntax element. */
+    uint8_t seq_parameter_set_id;
+    /** \brief Same as the H.264 bitstream syntax element. */
+    int32_t initial_cpb_removal_delay_nal[32];
+    /** \brief Same as the H.264 bitstream syntax element. */
+    int32_t initial_cpb_removal_delay_offset_nal[32];
+    /** \brief Same as the H.264 bitstream syntax element. */
+    int32_t initial_cpb_removal_delay_vcl[32];
+    /** \brief Same as the H.264 bitstream syntax element. */
+    int32_t initial_cpb_removal_delay_offset_vcl[32];
+    /**@}*/
+    /** @name sei picture timing*/    
+    /**@{*/    
+    /** \brief Same as the H.264 bitstream syntax element. */
+    int32_t cpb_removal_delay;    
+    /** \brief Same as the H.264 bitstream syntax element. */
+    int32_t dpb_output_delay;    
+    /** \brief Same as the H.264 bitstream syntax element. */
+    int32_t pic_struct;
```

Comment:
Missing timestamp fields?

Reply:
Remove this struct as SEI will be returned as bitstream.